pkg/sdn/plugin/node.go

```diff
@@ -248,12 +252,9 @@ func (node *OsdnNode) Start() error {
 			return true, nil
 		})
 
-	log.V(5).Infof("Creating and initializing openshift-sdn pod manager")
-	node.podManager, err = newPodManager(node.host, node.localSubnetCIDR, node.networkInfo, node.kClient, node.policy, node.mtu, node.oc)
-	if err != nil {
-		return err
-	}
-	if err := node.podManager.Start(cniserver.CNIServerSocketPath); err != nil {
+	// Kubelet has initialized, now we have a valid node.host
```

Comment:
@danwinship PTAL, I think we can just move the kubelet init block above the "everything is initilaized" comment; no reason it has to be that far down I don't think.